mattermost/mattermost-webapp
# MM-20336 add E2E for "Can change Custom Brand Image setting"

Summary
Test the ability to change "Custom Brand Image" setting

Test steps:

1. Login as a sysadmin and visit the page "/admin_console/site_config/customization" directly
2. Set EnableCustomBrand checkbox to true
3. Verify that setting is visible and matches text content
4. Verify that help setting is visible and matches text content
5. Upload the image
6. Verify that after page reload image exists

Ticket Link
Jira ticket MM-20336
closes https://github.com/mattermost/mattermost-server/issues/13166

Related Pull Requests
Related PR - #4210
